src/ServiceControlInstaller.Engine/Database/DatabaseMigrations.cs

```diff
@@ -4,14 +4,15 @@
     using System.Diagnostics;
     using System.IO;
     using System.Text;
+    using System.Text.RegularExpressions;
     using System.Threading;
     using ServiceControlInstaller.Engine.Instances;
 
     public class DatabaseMigrations
     {
         public static void RunDatabaseMigrations(IServiceControlInstance instance, Action<string> updateProgress)
         {
-            var timeout = (int)TimeSpan.FromMinutes(20).TotalMilliseconds;
+            var timeout = (int) TimeSpan.FromHours(12).TotalMilliseconds;
```

Comment:
Why?

Reply:
Remember an upgrade of the storage format on a 300 GB takes 4 hours roughly. I went for safety and added 12 h